pages/[...slug].vue

```diff
@@ -18,6 +18,39 @@
   <EpPageMeta :doc="doc" />
   <NuxtLayout>
     <div class="px-4 print:px-8 mt-10 mb-96 print:mb-0">
+      <div>
+        <nav>
+          <span v-if="breadcrumbs.length != 1">
+            <NuxtLink :to="'/'">
+              <Icon name="heroicons:home-20-solid" size="2em" />
```

Comment:
Icon is too large. It lacks borders like in task. It lacks signs that it is clickable.

![image](https://github.com/exactpro/docs-toolkit/assets/51794590/0a036b6f-9555-4767-a0ba-b400a332bdb8)

Reply:
there were no icons with borders when I searched for home , will revert the size back to default 1 em

Reply:
You should add borders using CSS on `NuxtLink`. And it would be preferable to add special styles on `hover`.